bonsai/ethereum-relay/src/uploader/pending_proofs/pending_proof_request_future.rs

```diff
@@ -110,10 +110,6 @@ impl Future for PendingProofRequest {
                             ) {
                                 ("SUCCEEDED", true) => Ok(receipt_response),
                                 ("SUCCEEDED", false) => {
-                                    // TODO: Should we consider this a failure
```

Comment:
There's also one below this that I added on the draft PR that was more stylistic and I forgot to remove, if you feel like removing another :) I can do it later if not

Reply:
Good point, @austinabell! 

I deliberately opted to keep this comment there for now as I agree with you that it should be refactored, but I consider that it'd be better to do it in a different PR. 

With regards to the actual effort put on refactoring, the Relay seems to be low priority for now so I'm not sure the effort would be the most recommended. Feel free to take it if you feel like it though! :)